v3/testdata/multDupeExts.pem

```diff
@@ -0,0 +1,133 @@
+                RSA Public-Key: (4096 bit)
+                Modulus:
+                    00:d4:bb:8b:0f:4e:f9:72:0d:08:19:c0:26:a4:e8:
+                    08:2f:33:8e:6c:b0:a9:dd:42:fc:cd:28:02:8a:45:
+                    ce:ac:0f:84:77:88:87:81:45:13:81:4e:c9:db:de:
+                    ad:bd:9f:08:66:60:be:a6:01:8f:dd:95:f9:ea:f3:
+                    5a:75:21:9d:13:42:18:64:b1:71:4a:5b:69:be:61:
+                    73:62:81:1e:85:e4:95:e5:59:14:a2:6d:82:2b:74:
+                    bd:3f:95:c6:0c:7a:71:8a:6b:d3:c5:01:4d:8e:c5:
+                    18:f1:56:53:8f:5f:15:f7:30:fe:64:07:31:6f:ef:
+                    8e:b9:51:8c:7f:db:18:c3:20:d7:a7:6c:14:d6:20:
+                    64:c3:ba:7a:ca:a0:3f:9e:06:41:e9:22:5f:22:33:
+                    9f:a5:91:ab:b6:85:96:e0:7a:f6:15:e9:10:58:92:
+                    3e:98:7d:30:f1:c5:58:92:20:09:d5:2f:dd:12:67:
+                    cd:0c:be:2f:c8:64:10:be:5e:40:62:0a:97:91:99:
+                    37:1c:c9:8a:b0:9d:f0:b3:5c:58:b7:1b:34:f7:bc:
+                    4c:04:0c:59:0a:8d:ce:c9:86:cf:29:75:9f:7d:d9:
+                    21:93:ec:f0:ce:4c:dd:1e:03:2a:d6:6c:3b:5d:96:
+                    cc:0e:60:fa:b9:3c:32:e0:27:a1:4c:5f:05:82:94:
+                    f5:4f:45:5c:c7:13:e5:70:d7:62:89:1b:19:b8:5a:
+                    6b:c5:40:24:07:d4:16:f6:29:89:92:02:f2:49:95:
+                    46:ae:18:38:e2:a6:59:32:96:74:e4:3c:20:8f:49:
+                    c1:de:9d:c0:00:42:4e:26:89:df:6f:c2:ea:d3:1a:
+                    2d:e6:f8:b8:56:c9:73:30:68:41:8d:49:91:ac:1b:
+                    6f:e5:f0:bd:f4:55:92:ba:54:f5:b8:58:de:d9:aa:
+                    22:b7:10:ae:c0:8d:ba:f4:5a:5e:8c:bd:af:4b:c1:
+                    96:0c:00:4e:91:45:86:a2:7a:82:90:30:84:6e:71:
+                    8f:a0:a0:27:65:32:27:52:a8:d5:85:2e:f2:8a:f4:
+                    15:76:86:d1:19:dc:b8:84:60:8b:33:d4:ec:91:0d:
+                    af:e7:3a:9c:79:5f:1f:66:3b:86:a9:47:62:ca:76:
+                    cd:ec:38:fb:c8:f5:b3:f4:9e:65:12:0a:7f:ae:25:
+                    25:54:59:f7:b5:02:ca:64:d4:9b:c3:f2:0f:3b:fe:
+                    80:af:7f:34:02:c8:da:7f:c3:f6:92:fd:6c:13:aa:
+                    b9:35:41:15:7e:29:73:7a:84:62:eb:5b:ba:9a:89:
+                    f5:84:b5:bf:0e:3c:ac:e6:cf:f5:61:61:97:ec:64:
+                    c9:b8:78:ab:a1:93:fa:05:fc:28:17:cb:aa:75:5b:
+                    41:6d:97
+                Exponent: 65537 (0x10001)
+        X509v3 extensions:
+            X509v3 Authority Key Identifier: 
+                keyid:EA:D3:9D:F2:FA:12:15:1D:6B:90:01:1F:1D:DB:27:7F:AF:D1:65:D7
+
+            X509v3 Subject Key Identifier: 
+                BC:93:A7:C1:4D:51:A1:B1:1E:5D:C9:C1:91:EA:DB:5B:53:D5:BC:58
+            X509v3 Key Usage: critical
+                Certificate Sign
+            X509v3 Basic Constraints: critical
+                CA:FALSE
+            X509v3 Subject Key Identifier: 
+                BC:93:A7:C1:4D:51:A1:B1:1E:5D:C9:C1:91:EA:DB:5B:53:D5:BC:58
```

Comment:
I took the certificate provided in https://github.com/zmap/zcertificate/issues/23 which already had a duplicated x509v3 Authority Key Identifier extension and then used [der-ascii](https://github.com/google/der-ascii/) to additionally duplicate the x509v3 Subject Key Identifier extension to make this test file.